kotlin/src/main/java/com/linecorp/armeria/common/kotlin/CoroutineContextUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.kotlin;
+
+import static java.util.Objects.requireNonNull;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.internal.server.annotation.AnnotatedService;
+import com.linecorp.armeria.server.kotlin.CoroutineContextService;
+
+import io.netty.util.AttributeKey;
+import kotlin.coroutines.CoroutineContext;
+
+/**
+ * Configures a coroutine context for annotated services and Kotlin suspending functions.
+ *
+ * @see CoroutineContextService
+ */
+public final class CoroutineContextUtil {
```

Comment:
How about `CoroutineContexts` unless there's a class with the same name in Kotlin?